custom_components/tuya_local/devices/lucci_fan.yaml

```diff
@@ -0,0 +1,23 @@
+name: Ceiling Fan
+products:
+  - id: U8wU126HQlIeQHvS
+    name: Lucci Connect WiFi Fan Remote
+primary_entity:
+  entity: fan
+  translation_only_key: fan_with_presets
```

Comment:
This translation key is not needed if there are no presets.

Reply:
removed